cachito/web/static/api_v1.yaml

```
@@ -277,6 +277,69 @@ paths:
                 $ref: "#/components/schemas/RequestConfiguration"
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/content-manifest":
+    get:
+      description: Return the content manifest of a request
+      parameters:
+      - name: id
+        in: path
+        required: true
+        description: The ID of the Cachito request to retrieve the content manifest for
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: The content manifest of the Cachito request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  metadata:
+                    type: object
+                    properties:
+                      icm_version:
+                        type: integer
+                        example: 1
+                      icm_spec:
+                        type: string
+                        example: "https://raw.githubusercontent.com/containerbuildsystem/\
+                                  atomic-reactor/f4abcfdaf8247a6b074f94fa84f3846f82d781c6/\
+                                  atomic_reactor/schemas/content_manifest.json"
```

I think this might be the preferred syntax for long strings in YAML:
```suggestion
                        example: >-
                          https://raw.githubusercontent.com/containerbuildsystem/
                          atomic-reactor/f4abcfdaf8247a6b074f94fa84f3846f82d781c6/
                          atomic_reactor/schemas/content_manifest.json"
```

---

Wouldn't this add spaces after the slashes though?

---

You're right